src/hooks/useToggle.ts

```diff
@@ -0,0 +1,10 @@
+import { useCallback, useState } from 'react'
+
+export function useToggle(defaultValue: boolean) {
+  const [value, setValue] = useState(defaultValue)
+  const onToggle = useCallback(() => {
+    setValue(!value)
+  }, [value])
+
+  return [value, onToggle] as [boolean, typeof onToggle]
```

Comment:
타입추론 잘 될텐데 따로 캐스팅 하신 이유 있을까용?🧐

Reply:
캐스팅을 하지 않는다면 아래 처럼 타입이 추론이 됩니다>

<img width="426" alt="스크린샷 2021-05-07 오후 4 16 21" src="https://user-images.githubusercontent.com/34743425/117413169-9b23ef00-af50-11eb-9f77-0ccd98492ef2.png">

```
() => toggleOptions 
```
<img width="310" alt="스크린샷 2021-05-07 오후 4 16 40" src="https://user-images.githubusercontent.com/34743425/117413166-99f2c200-af50-11eb-9bf3-42d24915aa7c.png">


```

Reply:
아하 그렇다면 타입 캐스팅 보단 타입을 선언 해주시는건 어떨까요?
```suggestion
  const result: [boolean, VoidFunction] = [value, onToggle]
  return result;
```

Reply:
https://github.com/Lubycon/clelab-frontend/pull/9/commits/46430937d8f665719ea89f4b0864b8f0c196c8db 해당 반영하였습니다.